README.md

```diff
@@ -1,2 +1,349 @@
+          * **patternId:** Unique identifier of the pattern (lower-case letters
+            without spaces)
+          * **level:** Severity level of the issue
+          * **category:** Category of the issue
+          * **parameters:** Parameters received by the pattern
+            * **name:** Unique identifier of the parameter (lower-case letters
+              without spaces)
+            * **default:** Default value of the parameter
+
+    ```json
+    {
+        "name":"jshint",
+        "version": "1.2.3",
+        "patterns":[
+        {
+            "patternId": "latedef",
+            "category": "ErrorProne",
+            "parameters": [
+            {
+                "name": "latedef",
+                "default": "nofunc"
+            }
+            ],
+            "level": "Warning"
+        }
+        ]
+    }
+    ```
+
+3. Write the code to run the tool
+
+  You can write the code in any language you want but, you have to invoke the
+  tool according to the configuration. After you have your results from the
+  tool, you should print them to the standard output in our Result format, one
+  result per line.
+
+* The filename should **not** include the prefix "/src/". Example:
+  * absolute path: /src/folder/file.js
+  * filename path: folder/file.js
+
+  ```json
+  {
+      "filename":"codacy/core/test.js",
+      "message":"found this in your code",
+      "patternId":"latedef",
+      "line":2
+  }
+  ```
+
+* If you are not able to run the analysis for any of the files requested you
+  should return an error for each one of them to the standard output in our
+  Error format.
+
+   ```json
+   {
+     "filename":"codacy/core/test.js",
+     "message":"could not parse the file"
+   }
+   ```
+
+* When an unknown error occurred while running the tool, it should send the
+  error information through the standard error and exit with error code 1. See
+  exit codes section.
+
+#### Tool Documentation
+
+At Codacy we strive to provide the best value to our users and, to accomplish
+that, we document our patterns so that the user can better understand the
+problem and fix it.
+
+At this point, your tool has everything it needs to run, but there is one other
+really important thing that you should do before submitting your docker: the
+documentation for your tool.
+
+Your files for this section should be placed in /docs/description/.
+
+In order to provide more details you can create:
+
+* A single /docs/description/description.json file.
+* A /docs/description/`<PATTERN-ID>.md` file for each pattern.
+
+##### Levels and Categories
+
+For level types we have:
+
+* Error
+* Warning
+* Info
+
+For category types we have:
+
+* ErrorProne
+* CodeStyle
+* Complexity
+* UnusedCode
+* Security
+* Compatibility
+* Performance
+* Documentation
+* BestPractice
+
+##### Description structure
+
+The documentation description is optional. So this could be skipped, but try as much as possible adding them.
```

Comment:
I would remove the "skipped" part. It's optional but the tool itself will bring way more value if the issue is properly documented

Reply:
Done ✅